### PR TITLE
1.9.0 small bug fixes

### PR DESCRIPTION
**Pull request by GhenadieVP**
A number of small bug fixes

- Proper unified Fungibles title display.
- Added additional info buttons:
  - For Presenting badges in TX Review.
  - For Customize Fees screen.
  - When scanning a connector extension QR code.
- Set the proper URLs tint color in the info sheet.

###  Presenting badge info
https://github.com/user-attachments/assets/06fed635-c1ab-4fd6-bf7f-9278bf8cd90e

### Fungibles display + ScanQR code info
https://github.com/user-attachments/assets/ffb9665b-09fe-4f06-aefa-5236967a2c92

### Customize fees info
https://github.com/user-attachments/assets/55fa993a-bf37-4b9a-816a-1780cad25ed3

### CODE REVIEW

File: RadixWallet/Features/AccountPreferencesFeature/Children/ThirdPartyDeposits/ResourcesList+View.swift [hunk 1]
```diff
@@ -149,7 +149,7 @@ private extension ResourceViewState {
 
 	var rowCoreViewState: PlainListRowCore.ViewState {
 		.init(
-			title: name ?? "-",
+			title: name,
```
Comment (GhenadieVP) [inline RadixWallet/Features/AccountPreferencesFeature/Children/ThirdPartyDeposits/ResourcesList+View.swift:152]: "-" is inferred from the ResourceBalanceView

File: RadixWallet/Features/AppFeature/Overlay/InfoLinkSheet+View.swift [hunk 2]
```diff
@@ -98,6 +98,7 @@ extension InfoLinkSheet {
 				Text(text)
 					.textStyle(.body1Regular)
 					.foregroundColor(.app.gray1)
+					.tint(.app.blue2)
```
Comment (GhenadieVP) [inline RadixWallet/Features/AppFeature/Overlay/InfoLinkSheet+View.swift:101]: Proper colour for Urls

File: RadixWallet/Features/DappsAndPersonas/DappDetails/DappDetails+View.swift [hunk 3]
```diff
@@ -290,22 +290,23 @@ extension DappDetails.View {
...
-							isOn: viewStore.binding(
-								get: \.isDepositsVisible,
-								send: { .depositsVisibleToggled($0) }
-							)
-						)
-
-						Separator()
-					}
-					.padding(.horizontal, .small2)
+					// TODO: Revert as part of 1.10.0 release
```
Comment (GhenadieVP) [inline RadixWallet/Features/DappsAndPersonas/DappDetails/DappDetails+View.swift:295]: Hide the account locker deposits functionality for the upcoming release

File: RadixWallet/Features/TransactionReviewFeature/TransactionReview+View.swift [hunk 4]
```diff
@@ -386,7 +386,6 @@ extension TransactionReview {
 			let proofsStore = store.scope(state: \.proofs) { .child(.proofs($0)) }
 			return IfLetStore(proofsStore) { childStore in
 				TransactionReviewProofs.View(store: childStore)
-					.padding(.bottom, .medium1)
```
Comment (GhenadieVP) [inline RadixWallet/Features/TransactionReviewFeature/TransactionReview+View.swift:389]: Removed extraneous padding.

File: RadixWallet/Features/TransactionReviewFeature/TransactionReviewProofs/TransactionReviewProofs+View.swift [hunk 5]
```diff
@@ -13,30 +13,23 @@ extension TransactionReviewProofs {
 
 		public var body: some SwiftUI.View {
 			WithViewStore(store, observe: { $0 }, send: { .view($0) }) { viewStore in
-				VStack(alignment: .leading, spacing: 0) {
+				VStack(alignment: .leading, spacing: .medium2) {
```
Comment (GhenadieVP) [inline RadixWallet/Features/TransactionReviewFeature/TransactionReviewProofs/TransactionReviewProofs+View.swift:16]: Added the info button + adjusted to proper paddings

File: RadixWallet/Features/TransactionReviewFeature/TransactionReviewProofs/TransactionReviewProofs+View.swift [hunk 6]
```diff
@@ -13,30 +13,23 @@ extension TransactionReviewProofs {
...
 					HStack {
 						Text(L10n.TransactionReview.presentingHeading)
 							.sectionHeading
 							.textCase(.uppercase)
 
-						//	FIXME: Uncomment and implement
-						//	TransactionReviewInfoButton {
-						//		viewStore.send(.infoTapped)
-						//	}
+						InfoButton(.badges)
```
Comment (CyonAlexRDX) [inline RadixWallet/Features/TransactionReviewFeature/TransactionReviewProofs/TransactionReviewProofs+View.swift:22]: 🚀

**Review by CyonAlexRDX — APPROVED** [review]
LGTM

**Review by danvleju-rdx — APPROVED** [review]

File: RadixWallet/Features/TransactionReviewFeature/CustomizeFees/CustomizeFees+View.swift [hunk 7]
```diff
@@ -158,7 +161,10 @@ extension CustomizeFees {
 				if viewState.insufficientBalance {
 					WarningErrorView(text: L10n.CustomizeNetworkFees.Warning.insufficientBalance, type: .error)
 				} else if viewState.linkingNewAccount {
-					WarningErrorView(text: L10n.TransactionReview.FeePayerValidation.linksNewAccount, type: .warning)
+					HStack(alignment: .center) {
```
Comment (matiasbzurovski) [inline RadixWallet/Features/TransactionReviewFeature/CustomizeFees/CustomizeFees+View.swift:164]: We probably want to set `spacing: .zero` here.

**Review by matiasbzurovski — APPROVED** [review]

**Pull request merged**